models/public/googlenet-v4-tf/freeze.py.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/research/slim/freeze.py b/research/slim/freeze.py
+new file mode 100644
+index 0000000..411001d
+--- /dev/null
++++ b/research/slim/freeze.py
+@@ -0,0 +1,62 @@
++import argparse
++import os
++import tensorflow as tf
++from tensorflow.python.framework import graph_io
++import subprocess
++
++from nets import nets_factory
++
++def parse_args():
++    """Parse input arguments"""
++
++    parser = argparse.ArgumentParser(description='Freeze models from TF Slim')
++
++    parser.add_argument('--ckpt', type=str, required=True,
++                        help='Path to .ckpt file')
++    parser.add_argument('--name', type=str, required=True,
++                        help='Model name')
++    parser.add_argument('--num_classes', type=int, required=True,
++                        help='Number of classes')
++    parser.add_argument('--output', type=str, required=False,
```

Comment:
Given that you specify the `--output` parameters both times you use this script, it can just be made mandatory. This will make some of the code unnecessary.

Alternatively, if the automatic output deduction works for both networks, you could remove the parameter instead.